cluster-up/cluster/images.sh

```diff
@@ -6,9 +6,10 @@ declare -A IMAGES
 IMAGES[gocli]="gocli@sha256:0cec3163e5f8b050f129fa795b76c20cb1eb27c91564c63663160b095064a1f9"
 if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
     IMAGES[k8s-fedora-1.17.0]="k8s-fedora-1.17.0@sha256:aebf67b8b1b499c721f4d98a7ab9542c680553a14cbc144d1fa701fe611f3c0d"
-    IMAGES[k8s-1.18]="k8s-1.18@sha256:f8bc8616139463a7e98330484f6c3748fe10c6ff5f5be614a373fd69b60a957c"
```

Comment:
question: why are the images for 1.16-1.18 changed as part of this commit? why would they be affected?

Reply:
Oh, wow, that's wrong! Thanks for the hint!

Reply:
its a bump, its ok that you take as `make bump-kubevirtci` gives
the hashes were changed and not bumped yet
correct me if i am wrong, but if i am not, you should take it